### PR TITLE
Publish 2021-03 version of the newsletter

### PR DESCRIPTION
**Pull request by Vinnl**

### CODE REVIEW

File: _posts/newsletter/2021-03-11-this-month-in-solid.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+permalink: /newsletter/2020-03-11
+tags: [monthly, updates]
+categories: [Updates]
+---
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+**11 March 2021** — It was exciting to see the great turnout at March’s Solid World. We continue to grow in the Solid community as reflected in Solid World’s highest attendance ever. During the event, we got to touch on a lot of great ideas related to the growing market of interoperability.
```
Comment (kay-kim) [inline _posts/newsletter/2021-03-11-this-month-in-solid.md:13]: heh ... from the content of the newsletter, not sure if they cover interoperability.  But eh.
Also, "growing market of interoperability" - not sure what that phrase means, but eh again.

**Review by kay-kim — APPROVED** [review]

File: _posts/newsletter/2021-03-11-this-month-in-solid.md (hunk 1 above)
Comment (Vinnl) [reply _posts/newsletter/2021-03-11-this-month-in-solid.md:13]: @MarrelleBailey Want to change this, or shall we just publish as-is?

Comment (MarrelleBailey) [reply _posts/newsletter/2021-03-11-this-month-in-solid.md:13]: Let's keep it. But I will keep a note of it

File: _posts/newsletter/2021-03-11-this-month-in-solid.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-03-11
+permalink: /newsletter/2020-03-11
```
Comment (Vinnl) [inline _posts/newsletter/2021-03-11-this-month-in-solid.md:4]: Whoops, wrong year again! I'll fix this right away as I don't think anyone would disagree with that ;)

```suggestion
title: This Month in Solid 2021-03-11
permalink: /newsletter/2021-03-11
```

**Pull request merged**